test/middleware.test.mo

```diff
@@ -34,11 +37,32 @@ await suite(
     await test(
       "when authToken has expired returns false",
       func() : async () {
-        let dayAgo : Time.Time = Time.now() - 90000000000000; // 25 hours ago in nanoseconds
         Map.set<Text, T.TokenWithTimestamp>(authTokenRegistry, Map.thash, authToken, (playerNftId, dayAgo));
         let response = Middleware.hasValidToken(playerNftId, authToken, authTokenRegistry);
         expect.bool(response).isFalse();
       },
     );
   },
 );
+
+await suite(
+  "#authTokenRegistryIsClean",
+  func() : async () {
+    await test(
+      "when cleaner is run old tokens are removed",
+      func() : async () {
+        Map.clear(authTokenRegistry);
+        Map.set<Text, T.TokenWithTimestamp>(authTokenRegistry, Map.thash, authToken, (playerNftId, Time.now()));
+        Map.set<Text, T.TokenWithTimestamp>(authTokenRegistry, Map.thash, authToken2, (playerNftId2, dayAgo));
+
+        let initialRegistrySize = Map.size(authTokenRegistry);
+        expect.nat(initialRegistrySize).equal(2);
+
+        Middleware.cleanAuthTokenRegistry(authTokenRegistry);
+
+        let finalRegistrySize = Map.size(authTokenRegistry);
+        expect.nat(finalRegistrySize).equal(1);
```

Comment:
Can you improve this equal? Instead of just be one, to be the younger one? I fear the younger one is being deleted.